composer.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "keboola/storage-driver-big-query",
+    "description": "Keboola Big Query driver",
+    "type": "project",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Roman Bracinik",
+            "email": "roman.bracinik@keboola.com"
```

Comment:
Nebude tady lepsi `Keboola developers` + `devel@keboola.com`? Nevim, jak je to jinde...

Reply:
nemame na to asi nejaku politiku davame tam len email toho co to zakladal https://github.com/keboola/php-storage-driver-teradata/blob/main/composer.json#L6-L11

Reply:
OK